gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -379,9 +379,9 @@ nnstreamer_query_server_data_new (void)
  * @brief free server handle
  */
 void
-nnstreamer_query_server_data_free (query_server_handle server_data)
+nnstreamer_query_server_data_free (query_server_handle * server_data)
```

Comment:
IMO, it looks not a standard way to use a handle. Such handling should be done in the caller.
For example, after calling `nnstreamer_query_server_data_free`, you may set `src->server_data = NULL`.

Reply:
Thanks! I think you're suggestion seems right. I updated the code, PTAL.